cli/args/mod.rs

```diff
@@ -1464,6 +1475,52 @@ pub fn load_env_variables_from_env_file(
     }
   }
 }
+pub fn load_env_variables_from_env_file_override(
+  filename: Option<&Vec<String>>,
+  flags_log_level: Option<log::Level>,
+) {
+  let Some(env_file_names) = filename else {
+    return;
+  };
+
+  for env_file_name in env_file_names.iter().rev() {
+    match from_filename_override(env_file_name) {
```

Comment:
Can't do this because it will overwrite env vars that existed when the application started, meaning `--watch` would have different behaviour than without `--watch`.